src/gen_dispatch.py

```diff
@@ -843,14 +864,17 @@ def write_source(self, f):
         self.outln('}')
         self.outln('')
 
+        """
```

Comment:
Please, don't comment out blocks: either code is used, or it's removed. We have sources under revision control for a reason.

Reply:
Removed. Please tell me if there is more improper code.